hoomd/trigger.py

```diff
@@ -38,11 +46,15 @@ def __init__(self, timestep):
         else:
             _hoomd.AfterTrigger.__init__(self, timestep)
 
+    def __str__(self):
+        return f"hoomd.trigger.After(timestep={self.timestep}"
 
 class Not(_hoomd.NotTrigger, Trigger):
     def __init__(self, trigger):
         _hoomd.NotTrigger.__init__(self, trigger)
 
+    def __str__(self):
+        return f"hoomd.trigger.Not(timestep={self.timestep}"
 
 class And(_hoomd.AndTrigger, Trigger):
```

Comment:
Should we give `And` and `Or` triggers a `__str__` method that does something like `hoomd.trigger.And([...])` where the ellipisis are replaced with the return string of `str(trigger)`?

Reply:
done

Reply:
Unfortunately, the objects accessed from the `triggers` list don't get the `__str__` method defined in Python:
```
'hoomd.trigger.And([<hoomd._hoomd.BeforeTrigger object at 0x7f8a29704d70>, <hoomd._hoomd.PeriodicTrigger object at 0x7f89f9da18b0>])'
```
We could solve this by implementing `__str__` in C++, but I'm not sure that it is worth it at this time.

Reply:
I wouldn't bother for this PR. It would be worth doing later though.